Detectors/TPC/simulation/macro/testHitsDigitsClusters.C

```diff
@@ -20,12 +33,15 @@
 
 using namespace o2::TPC;
 
+void drawSectorBoundaries();
+
 void testHitsDigitsClusters(int iEv=0,
-               std::string simFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/bin/AliceO2_TGeant3.tpc.mc_10_event.root",
-               std::string digiFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/bin/AliceO2_TGeant3.tpc.digi_10_event.root",
-               std::string clusFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/bin/AliceO2_TGeant3.tpc.clusters_10_event.root",
-               std::string trackFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/bin/tracks.root")
+               std::string simFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/AliceO2_TGeant3.tpc.mc_100_event.root",
+               std::string digiFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/AliceO2_TGeant3.tpc.digi_100_event.root",
+               std::string clusFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/AliceO2_TGeant3.tpc.clusters_100_event.root",
+               std::string trackFile="~/AliSoftware/sw/BUILD/O2-latest-O2dir/O2/tracks.root")
```

Comment:
I think this is also for testing. These are parameters and can be overwritten. @KlewinS can also comment.